cyphernodeconf_docker/templates/installer/docker/docker-compose.yaml

```diff
@@ -276,11 +276,14 @@ services:
       - .env/notifier.env
     volumes:
       - "<%= logs_datapath %>:/cnlogs"
+      - "<%= proxy_datapath %>:/proxy/db"
```

Comment:
Is this for pgpass?  Let's only mount that specific file... and not in /proxy/db but something more relevant.

Reply:
Yes.  Will do